exporter/prometheusremotewriteexporter/helper.go

```diff
@@ -222,10 +222,9 @@ func batchTimeSeries(tsMap map[string]*prompb.TimeSeries, maxBatchByteSize int)
 		sizeOfSeries := v.Size()
 
 		if sizeOfCurrentBatch+sizeOfSeries >= maxBatchByteSize {
-			wrapped := convertTimeseriesToRequest(tsArray)
-			requests = append(requests, wrapped)
+			requests = append(requests, &prompb.WriteRequest{Timeseries: tsArray})
 
-			tsArray = make([]prompb.TimeSeries, 0)
+			tsArray = tsArray[:0]
```

Comment:
Can you add a test for this? I am pretty sure that this will overwrite previous requests

Reply:
Yes, you are right. Sorry about the lack of quality of my pr. Close it.